src/app/views/home/sections/Mentors.js

```diff
@@ -17,24 +20,15 @@ function Mentors() {
 						Have hackathon experience and would love to share it with new
 						developers? Apply to be a mentor for ZotHacks 2022!
 					</p>
-					<Link
-						className="apply-button-wrapper"
-						to={{ pathname: "https://airtable.com/shrbPw3zLgnVZkMkA" }}
+					<a
+						href={MENTOR_APPLICATION_URL}
 						target="_blank"
+						rel="noreferrer"
 						data-place="bottom"
-						data-tip="Make sure to use your UCI email!"
+                        data-tip="Make sure to use your UCI email!"
 					>
-						<img
-							type="image"
-							className="apply-button"
-							style={{
-								height: "70px",
-								width: "auto",
-								marginTop: "0.5rem",
-							}}
-							src={mentorButton}
-						/>
-					</Link>
+						<button className="mentor-apply-button">APPLY TO BE A MENTOR</button>
```

Comment:
The button element should not be used for basic links. To apply button stylings, Bootstrap provides a `btn` class.